src/codeview.jl

```diff
@@ -94,13 +110,11 @@ function cthulu_typed(io::IO, debuginfo_key, src, rt, mi, iswarn, stable_code)
     println(io)
     println(io, "│ ─ $(string(Callsite(-1, MICallInfo(mi, rettype), :invoke)))")
 
-    if iswarn
-        cthulhu_warntype(io, src, rettype, debuginfo_key, stable_code)
-    elseif isa(src, IRCode)
-        Base.IRShow.show_ir(io, src; verbose_linetable = debuginfo_key === :source)
-    else
-        show(io, src, debuginfo = debuginfo_key)
-    end
+    iswarn && cthulhu_warntype(io, src, rettype, debuginfo, stable_code)
+
+    bb_color = (src isa IRCode && debuginfo === :compact) ? :normal : :light_black
+    irshow_config = IRShowConfig(lineprinter(src); bb_color)
+    show_ir(io, src, irshow_config)
```

Comment:
```suggestion
    if iswarn
        cthulhu_warntype(io, src, rettype, debuginfo, stable_code)
    else
        bb_color = (src isa IRCode && debuginfo === :compact) ? :normal : :light_black
        irshow_config = IRShowConfig(lineprinter(src); bb_color)
        show_ir(io, src, irshow_config)
    end
```
Cthulhu will print duplicated code otherwise ?

Reply:
Oops! Yeah, that's definitely not right.